client/lib/pages/main_pages/recent_numbers.dart

```diff
@@ -179,9 +179,8 @@ class _RecentNumbersPageState extends State<RecentNumbersPage> {
   Map<DataAggregation, Widget> _buildSegmentControlChildren(
       BuildContext context, DataAggregation selectedValue) {
     var valueToDisplayText = {
-      DataAggregation.total: S.of(context).latestNumbersPageTotalToggle,
-      // TODO: This string changed, so we must change regenerate translation keys someday.
       DataAggregation.daily: 'Daily',
```

Comment:
```suggestion
      // TODO: This string changed, so we must change regenerate translation keys someday.
      DataAggregation.daily: 'Daily',
```

Reply:
How will people find the TODO? It's essential to use a similar naming for the TODO as used elsewhere. All the translation TODOs I've seen have started `// TODO: Localize`. Start with that and then append something.
```suggestion
      // TODO: Localize - need to regenerate translation keys as this string was changed
      DataAggregation.daily: 'Daily',
```

Reply:
LGTM -- I just copied the old translation

Reply:
@theswerd I just put the old todo back